docs/reference/commandline/network_create.md

```diff
@@ -22,6 +22,7 @@ Create a network
 
 Options:
       --attachable           Enable manual container attachment
+      --ingress              Specify the network provides the routing-mesh
```

Comment:
We'll probably need some additional information to show why/how it's used; can you add a simple example in the "examples" section?